.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Ansible test
+on:
+  push:
```

Comment:
:crystal_ball: looking ahead

We will want to limit which kinds of pushes trigger integration tests when the time comes. It isn't necessary to run them on every push of _every_ branch. We probably only need on the default branch and at the end of pull requests.

Reply:
Agreed.